scripts/gen-dockerfile.sh

```diff
@@ -98,5 +123,3 @@ EXPOSE 20160 20180
 
 ENTRYPOINT ["/tikv-server"]
 EOT
-
-cat .gitignore > .dockerignore
```

Comment:
What about moving this line to the top of this file?

Reply:
Do you think it will always be the case that our .gitignore and .dockerignore will be the same?